### PR TITLE
Clarify PII concerns in the explainer.

### PR DESCRIPTION
**Pull request by snianu**
Closes https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/542

### CODE REVIEW

**Comment by snianu** [comment]
@caraitto Addressed your comment. PTAL.

File: docs/clipboard-pickling/explainer.md [hunk 1]
```diff
@@ -189,9 +189,9 @@ For #1 we need to update all browsers and convince web developers to migrate to
...
+This feature introduces custom clipboard formats with unsanitized content that will be exposed to both native apps and websites. Through the custom clipboard formats, PII may be transferable from web to native apps or vice versa. The content in the custom format is less visible/obvious to the users compared to the plain-text format so it might still be possible to transfer PII data without the knowledge of the user. This is also true for the existing [DataTransfer APIs](https://html.spec.whatwg.org/multipage/dnd.html#the-datatransfer-interface) that expose unsanitized HTML content in the standard HTML format(via setData/getData methods), but there may be metadata present in the custom format that wouldn't be typically included in the HTML format. The parsing rules for the custom format content and what data is included in the format, have to be defined by the native and web apps that read/write this format, so that alleviates some privacy concerns regarding who can read the sensitive data (if present) in the custom formats.
```
Comment (caraitto) [inline docs/clipboard-pickling/explainer.md]: nit: is less visible/obvious -> may be less visible/obvious.

**Review by caraitto — APPROVED** [review]
Thanks for making these changes!

File: docs/clipboard-pickling/explainer.md (hunk 1 above)
Comment (snianu) [reply docs/clipboard-pickling/explainer.md]: Done.

**Comment by snianu** [comment]
@travisleithead Need an LGTM from someone who has write access. Could you please take a look and let me know if you have any concerns?

**Comment by snianu** [comment]
@travisleithead Gentle ping..

**Comment by caraitto** [comment]
@travisleithead Hi, friendly ping?

**Review by BoCupp-Microsoft — APPROVED** [review]

**Pull request merged**